apps/gateway/src/users/users.service.ts

```diff
@@ -71,6 +71,11 @@ export class UsersService implements OnModuleInit {
   }
 
   async create(user: UserEntity | SignupUserDto) {
+    const existingUser = await this.findOne(user.email);
+    if (existingUser) {
+      throw new HttpException('Email already in use', HttpStatus.BAD_REQUEST);
```

Comment:
Kannst du vielleicht noch den Error wie wirs eben bei dem anderen besprochen hatten das durch die Translate Pipe jagen und das hier in ERRORS.EMAIL_ALREADY_IN_USE oder so ändern